expenses/config.py

```diff
@@ -1,5 +1,30 @@
 from google.cloud import storage
 import os
+from slack import WebClient
+from slack.errors import SlackApiError
 
 storage_client = storage.Client()
 bucket = storage_client.bucket(os.environ["GOOGLE_STORAGE_BUCKET"])
+
+slack_client = WebClient(token=os.environ.get('SLACK_API_TOKEN', ''))
+
+
+def send_slack_notification(requisition):
+    # Checks if slack token is provide as env var
+    if slack_client.token and requisition.created_by.slack_id:
+        if requisition.status == "Submitted":
+            message = f"Thank you for submitting requisition {requisition.headline}! You will receive alerts from me when the status is changed."
+        elif requisition.status == "Pending Changes":
+            message = f"Requisition {requisition.headline} ({requisition}) has requested changes. Notes by reviewer: {requisition.approval_set.latest('id').notes}"
+        else:
+            message = f"Requisition {requisition.headline} ({requisition}) status updated to *{requisition.status}*"
```

Comment:
not sure but I think linking the requisition reference string and putting the headline in parentheses, like:

> Requisition 2020-HGT7-01 (Streamers for the Klaus atrium)

might be a better format this.

But the main thing is that linking the requisition reference string would be really helpful

Reply:
Yeah the {requisition} prints the reference string, I can change the format to that though, bc its reversed